src/common/Loader/Loader.tsx

```diff
@@ -1,9 +1,22 @@
 import React from "react";
 
-export const DataSpinner = () => {
+export const DataSpinner = ({
```

Comment:
This is a common loader that we consume on all places. Passing on these props do not make sense.
- Instead of `isSmallScreen`, we can go with something like this: `size` which can be xs, sm, md, lg etc.
-  `isMemberSelector` is odd because why would a loader would have a prop like that. If you need something that's very specific to loader that's being consumed for MemberSelector, you can create another local component specific for that.

Reply:
fixed